src/renderers/additional/categorization-renderer.tsx

```diff
@@ -1,12 +1,89 @@
 import { JSX } from '../JSX';
 import * as _ from 'lodash';
 import { JsonForms } from '../../core';
-import { Renderer, RendererProps } from '../../core/renderer';
+import { RendererProps } from '../../core/renderer';
 import { and, RankedTester, rankWith, uiTypeIs } from '../../core/testers';
 import { Categorization, Category } from '../../models/uischema';
 import { mapStateToLayoutProps, registerStartupRenderer } from '../renderer.util';
 import DispatchRenderer from '../dispatch-renderer';
 import { Component, connect } from '../../common/binding';
+import {JsonSchema} from "../../models/jsonSchema";
+
+const getCategoryClassName = (category: Category, selectedCategory: Category): string =>
+  selectedCategory === category ? 'selected' : '';
+
+export interface CategorizationProps {
```

Comment:
why do we need to export all those props and inner components?

Reply:
As discussed, we'll keep these in case anyone would create custom renderers for categories.